libkbfs/tlf_handle.go

```diff
@@ -742,40 +381,51 @@ func getSortedUnresolved(unresolved map[keybase1.SocialAssertion]bool) []keybase
 	return assertions
 }
 
-func splitAndNormalizeTLFName(name string, public bool) (
-	writerNames, readerNames []string,
-	extensionSuffix string, err error) {
-
+func splitTLFName(name string) (wnames []string, rnames []string, ext string, err error) {
```

Comment:
Done, except it *can* return an error on the second splitN, so just handling that error properly.